libmscore/rendermidi.cpp

```diff
@@ -276,7 +276,7 @@ static void playNote(EventMap* events, const Note* note, int channel, int pitch,
                         easeInOut.timeList(static_cast<int>((timeDelta + timeStep * 0.5) / timeStep), int(timeDelta), &onTimes);
                         double nTimes = static_cast<double>(onTimes.size() - 1);
                         for (double time : onTimes) {
-                              int p = int(pitch + (t / nTimes) * pitchDelta);
+                              int p = int((t / nTimes) * pitchDelta);
```

Comment:
I am wondering what is the likelihood that the nTimes variable will be 0 here

Reply:
Good point. I will make sure this is robust.

Reply:
Actually, EaseInOut::timeList(...) will always produce at least one time so the likelyhood that nTime will be 0 is 0.

Reply:
Isn't `nTimes` one less than than the length of the time list?

Reply:
Oh yes. You're right. I was going to commit a change that did trap both conditions anyways.

Reply:
And add the std::abs I added in PR #7015 that somehow disapeared here.

Reply:
I checked and nTimes should always be at least 3 whatever the timeDelta and assuming pitchDelta cannot be less than one half step.